app/src/main/res/layout/card_new_app_version.xml

```diff
@@ -0,0 +1,233 @@
+    android:id="@+id/new_package_card"
+    android:layout_width="match_parent"
+    android:layout_height="240dp"
+    android:layout_marginBottom="5dp"
+    android:layout_marginEnd="8dp"
+    android:layout_marginLeft="8dp"
+    android:layout_marginRight="8dp"
+    android:layout_marginStart="8dp"
+    android:layout_marginTop="15dp"
+    app:cardCornerRadius="4dp"
+    style="?attr/backgroundCard"
+    >
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      >
+
+    <ImageView
+        android:id="@+id/app_background_image"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:adjustViewBounds="true"
+        android:scaleType="centerCrop"
+        tools:src="@drawable/aptoide_gradient"
+        />
+
+    <View
+        android:id="@+id/lower_background"
+        android:layout_width="match_parent"
+        android:layout_height="120dp"
+        android:background="?attr/backgroundCardColor"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        />
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="100dp"
+        android:layout_marginTop="60dp"
+        android:background="@drawable/editorial_bottom_up_black_gradient"
+        app:layout_constraintBottom_toTopOf="@+id/lower_background"
+        />
+
+    <ImageView
+        android:id="@+id/app_icon"
+        android:layout_width="64dp"
+        android:layout_height="64dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="52dp"
+        app:layout_constraintBottom_toTopOf="@+id/lower_background"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/lower_background"
+        tools:src="@color/red"
+        />
+
+    <TextView
+        android:id="@+id/app_name"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="15dp"
+        android:layout_marginStart="15dp"
+        android:layout_marginTop="5dp"
+        android:ellipsize="end"
+        app:layout_constraintStart_toEndOf="@id/app_icon"
+        app:layout_constraintTop_toTopOf="@id/lower_background"
+        tools:text="LEAGUE DAS LEGENDAS"
+        style="@style/Aptoide.TextView.Regular.M.BlackAlpha"
+        />
+
+    <TextView
+        android:id="@+id/version_text"
+        android:layout_width="150dp"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="46dp"
+        android:layout_marginLeft="15dp"
+        android:layout_marginStart="15dp"
+        android:lines="1"
+        android:text="@string/promotional_new_version"
+        android:textColor="@color/aptoide_orange"
```

Comment:
Why not use a textview style with the correct color?

Reply:
Done, created aptoide_orange style